### ISSUE TITLE
Request Week from datetime

### ISSUE THREAD

**Issue by jperez999** [rapidsai/cudf#4908]
Currently cudf allows for splitting a date time into year, month and day. Like this:
```
    # Expand date into [Year, Month, Day]
    df['Date'] = df['Date'].astype('datetime64[s]')
    for attr in ['Year', 'Month', 'Day']:
        df[attr] = getattr(df.Date.dt, attr.lower())

```
Would like to support the Week property also, like in pandas:
```
   # Expand date into [Year, Month, Day, Week]
    df.Date = pd.to_datetime(df.Date)
    for attr in ['Year', 'Month', 'Day', 'Week']:
        df[attr] = getattr(pd.DatetimeIndex(df.Date), attr.lower())
```

**Comment by rnyak** [comment]
@harrism is there any update on that FEA? Thanks.

**Comment by harrism** [comment]
What format of week do you need? Just the week number (0-51, or 1-52)?

**Comment by rnyak** [comment]
@harrism, the week number 1-52. Pandas generates the `week`  column as `int64`. Thanks.

**Comment by beckernick** [comment]
For Python bindings, note that pandas is deprecating the `series.dt.week` API in favor of `series.dt.isocalendar().week` as per https://pandas.pydata.org/pandas-docs/stable/whatsnew/v1.1.0.html#deprecations

**Comment by rnyak** [comment]
@kkraus14 are we going to have this support in 0.19 release? thanks.

**Comment by kkraus14** [comment]
> @kkraus14 are we going to have this support in 0.19 release? thanks.

No, unfortunately there wasn't bandwidth to tackle this in 0.19. Moved to 0.20.

**Comment by shwina** [comment]
I think this issue can be closen given that (1) the `isocalendar` API is available in cuDF starting v21.12 (currently, that's available in the nightly build), and (2) Pandas has deprecated the `.dt.week` in favor of `isocalendar`.

**Comment by beckernick** [comment]
Closing as completed and deprecated 👍
